src/components/AccountLoginForm.jsx

```diff
@@ -26,6 +26,7 @@ const AccountLoginForm = ({ t, isOAuth, customView, fields, error, dirty, submit
               name={inputName}
               placeholder={t('account.connection.login.password.placeholder')}
               invalid={!!error}
+              noPasswordSave
```

Comment:
I see that `noAutoFill` is really a way to control the `autocomplete` attribute of the underlying `input`. It seems to me simpler to use `autocomplete` since web-developers are familiar with this API and will automatically what it means.

Kinda relevant XKCD : https://xkcd.com/927/